README.md

```diff
@@ -463,7 +469,26 @@ Behind the scenes, the OpenTelemetry Operator will convert the Collector’s con
           exporters: [logging]
 ```
 
-Note how the Operator added a `global` section and a new `http_sd_configs` to the `otel-collector` scrape config, pointing to a Target Allocator instance it provisioned.
+Note how the Operator removes any existing service discovery configurations (e.g., `static_configs`, `file_sd_configs`, etc.) from the `scrape_configs` section and adds a single `http_sd_configs` configuration, pointing to a Target Allocator instance it provisioned.
```

Comment:
```suggestion
Note how the Operator removes any existing service discovery configurations (e.g., `static_configs`, `file_sd_configs`, etc.) from the `scrape_configs` section and adds an `http_sd_configs` configuration pointing to a Target Allocator instance it provisioned.
```

More than one `http_sd_configs` entry will be present if there is more than one scrape job in the incoming config.  There should be one per job.  The example only had one job, but a real config is likely to have more.

Reply:
@Aneurysm9 The replacement process occurs for each job in the incoming configuration. The code iterates through all the jobs and updates the configuration accordingly. If it would be helpful for clarity, I can include an additional job in the example.